breathecode/marketing/tests/urls/tests_lead.py

```diff
@@ -0,0 +1,521 @@
+                'academy': 1,
+                'user': None,
+                'tag_objects': [],
+                'automation_objects': []
+            })
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.assertEqual(self.all_form_entry_dict(),
+                         [{
+                             'id': 1,
+                             'contact_id': None,
+                             'fb_leadgen_id': json['fb_leadgen_id'],
+                             'fb_page_id': json['fb_page_id'],
+                             'fb_form_id': json['fb_form_id'],
+                             'fb_adgroup_id': json['fb_adgroup_id'],
+                             'fb_ad_id': json['fb_ad_id'],
+                             'first_name': json['first_name'],
+                             'last_name': json['last_name'],
+                             'email': json['email'],
+                             'phone': json['phone'],
+                             'course': json['course'],
+                             'client_comments': json['client_comments'],
+                             'location': json['location'],
+                             'language': json['language'],
+                             'utm_url': json['utm_url'],
+                             'utm_medium': json['utm_medium'],
+                             'utm_campaign': json['utm_campaign'],
+                             'utm_source': json['utm_source'],
+                             'referral_key': json['referral_key'],
+                             'gclid': json['gclid'],
+                             'tags': json['tags'],
+                             'automations': json['automations'],
+                             'street_address': json['street_address'],
+                             'country': json['country'],
+                             'city': json['city'],
+                             'latitude': float(json['latitude']),
+                             'longitude': float(json['longitude']),
+                             'state': json['state'],
+                             'zip_code': json['zip_code'],
+                             'browser_lang': json['browser_lang'],
+                             'storage_status': json['storage_status'],
+                             'lead_type': json['lead_type'],
+                             'deal_status': json['deal_status'],
+                             'sentiment': json['sentiment'],
+                             'academy_id': 1,
+                             'user_id': None,
+                             'ac_contact_id': json['ac_contact_id'],
+                             'ac_deal_id': json['ac_deal_id'],
+                             'won_at': json['won_at']
+                         }])
+
+    @patch(GOOGLE_CLOUD_PATH['bucket'], apply_google_cloud_bucket_mock())
+    @patch(GOOGLE_CLOUD_PATH['blob'], apply_google_cloud_blob_mock())
+    def test_lead__with__data_active_campaign_slug_priority(self):
```

Comment:
Each HTTP method who doesn't a GET is better represent it in the method name, some like `test_lead__post__with_data_active_campaign_slug_priority`